src/tools/rbd_mirror/image_replayer/snapshot/Replayer.cc

```diff
@@ -1102,8 +1106,12 @@ void Replayer<I>::handle_copy_image(int r) {
 
   {
     std::unique_lock locker{m_lock};
+    m_last_snapshot_bytes = m_snapshot_bytes;
     m_bytes_per_snapshot(m_snapshot_bytes);
+
     auto time = ceph_clock_now() - m_snapshot_replay_start;
+    m_last_snapshot_mseconds = time.to_msec();
```

Comment:
Since in the end of the day seconds are reported, why not just use `sec` accessor instead of performing `to_msec` calculation and than manually dividing by 1000?